clients/js/packages/chromadb-core/src/embeddings/CohereEmbeddingFunction.ts

```diff
@@ -66,13 +67,22 @@ class CohereAISDK7 implements CohereAIAPI {
   public async createEmbedding(params: {
     model: string;
     input: string[];
+    isImage?: boolean;
```

Comment:
I am assuming input is supposed to be a base64 encoded png if isImage is true? 

Could we put that in a comment?